src/web/theme/pages/OrderReference/EnhanceOrderReference.js

```diff
@@ -0,0 +1,49 @@
+import { graphql } from "react-apollo";
+import compose from "recompose/compose";
+import withState from "recompose/withState";
+import withHandlers from "recompose/withHandlers";
+
+const orders = [
+  { id: 468513544, products: [{ sku: "epi_guitar" }] },
+  { id: 123456789, products: [{ sku: "epi_guitar" }, { sku: "epi_guitar02" }] },
+  {
+    id: 987654321,
+    products: [
+      { sku: "epi_guitar" },
+      { sku: "epi_guitar02" },
+      { sku: "epi_guitar" },
+      { sku: "epi_guitar02" }
+    ]
+  },
+  {
+    id: 654987321,
+    products: [
+      { sku: "epi_guitar" },
```

Comment:
Is it voluntary to have duplicate skus everywhere ?

Reply:
yes :) more guitars for my band !